### PR TITLE
Adding EE Basic Support Dump

### PR DESCRIPTION
**Pull request by yongshin**
Adding support dump for EE engine only

### CODE REVIEW

File: ee/get-support.md [hunk 1]
```diff
@@ -45,6 +45,33 @@ This support dump only contains logs for the node where you're running the
...
+
+```
+docker run --rm  \
+    -v /boot:/boot \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /var/run/docker.pid:/var/run/docker.pid:ro \
+    -v /var/run/docker:/var/run/docker \
+    -v /var/lib/docker:/var/lib/docker \
+    -v /var/run/docker/netns:/var/run/docker/netns \
+    -v /var/lib/docker:/var/lib/docker \
```
Comment (ChristianKniep) [inline ee/get-support.md]: `/var/lib/docker` is provided twice

Comment (yongshin) [reply ee/get-support.md]: Thanks! Just removed the duplicate line.

**Comment by dani-docker** [comment]
> @dani-docker: see any reason to add conditionals and a more complex support dump cli formulation to the "get support" page?

@trapier I ran into the same issue you referred to here 
https://github.com/docker/orca/issues/14624#issue-354509259 hence the suggestion/workaround is to deploy dsinfo directly. 
But I agree, fixing the issue here https://github.com/docker/orca/issues/14624 is more appropriate.

@yongshin what do you think?

**Comment by yongshin** [comment]
Yea lets just go ahead and fix that issue instead.

**Comment by trapier** [comment]
@yongshin: sounds great.  Can you close this pr?

**Pull request closed without merging**